Instruments/microwave_tuning_gui.py

```diff
@@ -81,8 +82,8 @@ def on_about(self):
         QMessageBox.about(self, "About the demo", msg.strip())
     
     def orig_zoom_limits(self):
-        for ini_val, w in [('9700000',self.textbox1),
-                ('9900000',self.textbox2)]:
```

Comment:
this is way too zoomed out and may result in jumping over the actual dip with only 15 points

Reply:
OK, that's reasonable -- it's worthwhile to add a comment on the typical location and width of the dip, then

Reply:
after rerunning and testing these are the final values that produce the dip shown on slack (https://jmfrancklab.slack.com/archives/CLMMYDD98/p1705090019740609)